### PR TITLE
[SM64] Fix #421 - Custom path DL/Geolayout exports

### LINKED ISSUE #421: Can't export object's using custom export path
error: cannot access local variable 'scrollName' where it is not associated with a value

### PR DESCRIPTION
**Pull request by ThePerfectMario64**

### CODE REVIEW

**Comment by ThePerfectMario64** [comment]
fixes #421

**Comment by Lilaa3** [comment]
As I guessed this is another combined exporter regression but I'm ok with merging this seperately, good catch

File: fast64_internal/sm64/sm64_f3d_writer.py [hunk 1]
```diff
@@ -377,6 +377,8 @@ def sm64ExportF3DtoC(
         scrollName = "actor_dl_" + name
     elif headerType == "Level":
         scrollName = levelName + "_level_dl_" + name
+    elif headerType == "Custom":
+        scrollName = "actor_geo_" + os.path.dirname(name)
```
Comment (Lilaa3) [inline fast64_internal/sm64/sm64_f3d_writer.py:381]:
```suggestion
    elif headerType == "Custom":
        scrollName = "dl_" + name
```

File: fast64_internal/sm64/sm64_geolayout_writer.py [hunk 2]
```diff
@@ -627,6 +627,8 @@ def saveGeolayoutC(
         scrollName = "actor_geo_" + dirName
     elif headerType == "Level":
         scrollName = levelName + "_level_geo_" + dirName
+    elif headerType == "Custom":
+        scrollName = "actor_geo_" + os.path.dirname(dirName)
```
Comment (Lilaa3) [inline fast64_internal/sm64/sm64_geolayout_writer.py:631]:
```suggestion
    elif headerType == "Custom":
        scrollName = "geo_" + dirName
```

**Review by Lilaa3 — APPROVED** [review]
Good catch and thank you for contributing, I have other regressions adressed in my own pr so if you spot anymore you can check if it's been fixed in it

**Pull request merged**